src/api/app/models/user_ldap_strategy.rb

```diff
@@ -37,7 +37,7 @@ def self.find_group_with_ldap(group)
   end
 
   # This static method performs the search with the given search_base, filter
-  def self.search_ldap(search_base, filter, required_attr = nil)
```

Comment:
Why it's not used, I can't say. The [first](b8d3f416ac18d5563f87d5c29fd73498320e407e) commit introducing it, was already not using it. Obviously it was never needed.

I'll add this to the commit message